src/lib/token_list_gen.ts

```diff
@@ -285,7 +289,31 @@ export const generateTokenList = async (
     if (uri) logoUris[token.l1TokenAddr] = uri;
   }
 
-  let arbifiedTokenList: ArbTokenInfo[] = tokens
+  const getL1GatewayAddress = async (l2GatewayAddress: string) => {
+    let l2Gateway: string;
+    if (isNova) {
+      if (
+        (l2Gateway = l2ToL1GatewayAddressesNova[l2GatewayAddress.toLowerCase()])
```

Comment:
is this line supposed to be making an assignment to `l2Gateway` or a conditional statement?

Reply:
yeah - assignment + pass truthy conditional if in mapping, assign undefined so falsy if not in mapping